Integrations/integration-Cisco-umbrella.yml

```diff
@@ -1,16 +1,16 @@
18JmsRlGefyq/nhYSYuQTOtQz9IwG6+EEyHNhuIK3D5DRfY6ZP0yDkk17p+rsrlks15JZnXY3Gieg80QxmthY2Xfyuy8uz7owj3H/zz5qV97SJqOsyrpu3kVZdfBN3RL++TH7s1tv+Xu0nKhAxjAaUQpCwC4lfIwa5cyzdul6+ra5LvmU38563oAxUOX7vKM5/+X3fe89yuSvXv3zqosTOq1Jlc0OWAzJo80JJtF2/E1KbtuwUMH4MX82N5f2S3czoQes145CpNsZVZPp6EcdUvdPJvTzXEIl172VYee8LS/S178un+2mXbdePOqbq8PYTTv0BuTeFzmnZ3mbZ46w/VD6lvs7AwAAHzBk0D7yXz75/LkFku1vJR5F8zqWajTabuVcZZn+//dj/746tc84I3JOXfdMyvSkJZVkxaSsN7kuirvMuEo22W3397cXXVL2ndXHnJy7LxbprGjcszTVZg0a0xiGp9WYVwmTeZu0lu50lza5L/R7nFy6NDjLtj7tPdM/u/rL5kmsXty1hcn6cbshnHCldAtQ74HAADOIH3gq/p/6TBPr5+3lyZ5mwtj1gtVMQ+FST5Jk1e87fJD9bnvTw7e/as21w6szrI8VNMTTT4oQ5ithyzvqqOXyXA3cX5hsrRw+8zqMErGKyFLmxxXxDHWqs1/aZPNmpgddiVJlW0UYaPZ18Xe/avVhRfd+9DFR9+x+9/8/KNj6YtQjZonZjT/DqAORbuUSdE+b3GItQzdmF03zlfNvyswfxMAAO4YPmkDAMBpZqMO9YF/ds1Pjm/42DOTUOexynQ8mjh2i43dYYcji+Pp8HMsYhErTsfrucMldV0nk8mkPROfn9jVKZquHz/30O++6rXhq7/+od1z1FVkHzhYCTjDjKpKWzoAuL2aKBfitLa73P0BLxl//G++Z4gGQ5feId8Nl8VcN3SO5Y4V81pcppsbo/F4XA+5OubvoXtyubm599CbX/balYuueUD/ZAVPDQAAnI0BInTHtsSS9H1B83Ov+OrnTz74vucPWWLIGEOuG/Ke8bs73jBWGsfw4j4fMveQt4eM3Xbzvfno/S5409Nes3ro+y4o6zIkqSmUAABwtimrLqcNeaCNDF/7pPtd8Mr/+Lpq/cRd4hjRkCtizotZIp7WVSVA3NHxO+7nZr8P82b/f/buBM62qy7w/X+ttfc+59R86w51EzKQkDCIYYYQCQHEEBBlCAHsZkYGRWZUsIE2KvJ4ODSIA41NQ9sO8Hj9FMeW97TbVj/YTq2NPhVoEVEGwxBCcm+ds/da663/2nufOvcSeFbVqcqte3/f8n/PWJXPZ+/1wfXfa+3/X/X5nebXwXur39H1PD1P+bMYF47+j/f//OD+1z+GIwgAAACcgfN8DgEAAACwj2IhjZhc2VkrOmuHINNVAq9NIcYUKxc99SUfKA4ffaksHV4WZ03ZBEkRXSikiKX49D2NKLq5Kv21OBETtMhF/n2O8e5PUo7SS47CO7HR5Rrttd7k5ILURZQmjFNG5aXQDYuNFmcfpl8bSLN+4QXnLS/97sqzvvfJm/nPtWXYcyfeGKcdnXKZ72DFhZgrfDe2DQC4Y//Xb8vx9n+6AADAV9W0kXK1JvR3Nm11dsq5Xh3lgm950f9t7n6vV48Ha+aEGZjGpvwt5QJVSg7KMNNhNjRSpiQx5JtpNGUIOaZ/D3NV+jqHVKWMozVVrNL5SOcy/ehxN2YgMY6kPnzhnc+7/7E/Hj719Y8bm/as51xedzY2TfuYzlnf2Sma0EX3H+qGhf6OidOXAAAAAPZTLNqQU9fSTBf54mjQCbtPz798xh5MmvuXmg+kfE4Kc/5zXvp/lBee/wM+LovXtSLrc2huYNPE3zRBnOZ5uh5oqBm8Wy6YLtr8uL+e3edfXrvt5jxaz0Gbp+WeuiHk9bnClNH4SiYp74tuVZoLL7jqvKtv+n+HF/xvd2s0z9NzaGzu2qzncPof0DHB6QMAAAAOlH4fXr8vz3Tzel23Ce0Cj7j8Rk4ixNwo1j3vLQ8/b+T+ezh0weU+LIhxC1rxIuUVKT8wk5QzjNt9g85Jbcr0F8r0913O+8r09zUwH5pj5yhCzsXLlKNpOK/HuUinLH2W10/HUha6/zblhKZ0fv389dWN5V9ZfcJrX6K1RzZFc0c9v6XVaF+FU64HSH89IJ8+K9xiBwAAAOwNZtoAAADAPjNbl8FbM92Yjl75sLe5Wz57rVaRThG1krRzuqTS7aOLLHocBFoVfO3Dv//20d2+6aH5zHVrIHpeY/cj/bnU6uG09QVwBtD/JZrMvP60yEmOCgAA/wzd3N5Z1871Z9I8/ehO51/0VnfL5x5pu85OfQchHID8PZ2rvqOTbxp36G9/798OL3/cg93s6lrfwbfv9hvDVt4HAAAA4KAlAdN1m5iL1sWZqb/Nyzn68aG7PvQH7Gf+8ckhvaE5Q5/v4cyVzpVJ5ylqnjeZtFfCw2SyvlG9/z+U5knn+W6pTnO5U/J21vEAAACAA8lrkbp+bq+PXX7X7900fR6n0/33vuCy4x96/8/4uh6xjnfwLX7mz75v8Yp/8TTT5Xhy49aiXYjkdwAAAMAdgZU0AAAAYJ+5PBGvRXyTO/sE67Tjrjv/yc97uxy54Jm32oGMB0PxdTDOFBJLZ8b6vWjT90qR0EbIlcJFKh+kCnV67nNgl+cntjFtmTvtmdv15opljiBFG7qvUbT698kc3rvc0Uk2Llw/79LFD5hvf8fVRpt05T8TRctZ5JIWhZWxdvDVCu96nmWS3m84AQDuMFq5YsBhAABge0yRezf53AXWp1yv+ylysyY5esMr3mmvfMRLN8vCbMZGgp+I006vKafQrk21jTmaLrSLUO4k1OUh6a0cvWBMDu0cFdlLt2t9Hl35NqKUOcqmjSLWUqXPB0UZYqikPn7B8fPudfi/lv/qP1051uOvPZx0s6M+phPig3ZgtjPFLNgQBwAAAJzpYv+Pr9s5vBYusLomlxI743Lf
```

Comment:
Ido noted on some slack channel that all integrations from the same vendor should be with the same image (Cisco Umbrella & Cisco Meraki for example)